src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -129,14 +137,14 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
                 this.closeModal();
                 return;
             }
+        }
 
-            if (this.profileToOnlineMods.length === 0) {
-                this.activeStep = 'NO_PACKAGES_IN_IMPORT';
-                return;
-            }
-
-            this.activeStep = 'REVIEW_IMPORT';
+        if (this.profileToOnlineMods.length === 0) {
```

Comment:
Nitpick: could be onelined:
`this.activeStep = this.profileToOnlineMods.length ? 'REVIEW_IMPORT' : 'NO_PACKAGES_IN_IMPORT';`